joblib/test/test_pool.py

```diff
@@ -406,10 +391,13 @@ def test_memmaping_on_dev_shm():
         # pickling procedure generate a .pkl and a .npy file:
         assert_equal(len(os.listdir(pool_temp_folder)), 2)
 
-        b = np.ones(100, dtype=np.float64)
+        # create a new array with a content that is different from a so that
```

Comment:
maybe from 'a' (i.e. quotes around a) to make it clearer that you are talking about the 'a' variable and not the 'a' article.